docs/logagent/output-filter-dropevents.md

```diff
@@ -0,0 +1,38 @@
+title: Drop Events Filter 
+description: Drop events before logs are shipped to any destination
+
+## Output filter: drop-events
+
+This plugin drops events by include/exclude criteria for each field. The filters property contains a list of fields. For each field, you can specify a regular expression to keep (include) or drop (exclude) the event from processing.
```

Comment:
The filters property contains a list of fields.
=>
The `filters` property contains a list of fields. 

Make sure filters is surrounded by backticks so it has the context of the property, not the actual word "filters".